### PR TITLE
zh-trans: add /content/zh/docs/tasks/administer-cluster/namespaces-wa…

### PR DESCRIPTION
**Pull request by yyqqing**

### CODE REVIEW

**Comment by chenrui333** [comment]
This is what exists in the previous release:
https://v1-12.docs.kubernetes.io/zh/docs/tasks/administer-cluster/namespaces-walkthrough/

File: content/zh/docs/tasks/administer-cluster/namespaces-walkthrough.md [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+有助于不同的项目、团队或客户去共享 Kubernetes 集群。
+
+<!-- It does this by providing the following:
+
+1. A scope for [Names](/docs/concepts/overview/working-with-objects/names/).
+2. A mechanism to attach authorization and policy to a subsection of the cluster. -->
+
+名字空间通过以下方式实现这点：
+
+1. 为[名字](/docs/concepts/overview/working-with-objects/names/) 设置作用域.
```
Comment (chenrui333) [inline content/zh/docs/tasks/administer-cluster/namespaces-walkthrough.md]:
```suggestion
1. 为[名字](/docs/concepts/overview/working-with-objects/names/)设置作用域.
```

File: content/zh/docs/tasks/administer-cluster/namespaces-walkthrough.md [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+<!-- The development team would like to maintain a space in the cluster where they can get a view on the list of Pods, Services, and Deployments
+they use to build and run their application.  In this space, Kubernetes resources come and go, and the restrictions on who can or cannot modify resources
+are relaxed to enable agile development. -->
+
+开发团队希望在集群中维护一个空间，以便他们可以查看用于构建和运行其应用程序的 Pods、Services 和 Deployments 列表。在这个空间里，Kubernetes 资源被自由地加入或移除，对谁能够或不能修改资源的限制被放宽，以实现敏捷开发。
+
+<!-- The operations team would like to maintain a space in the cluster where they can enforce strict procedures on who can or cannot manipulate the set of
+Pods, Services, and Deployments that run the production site. -->
+
+运维团队希望在集群中维护一个空间，以便他们可以强制实施一些严格的规程，对谁可以或不可以操作运行生产站点的 Pods、Services 和 Deployments 集合进行控制。
```
Comment (chenrui333) [inline content/zh/docs/tasks/administer-cluster/namespaces-walkthrough.md]:
```suggestion
运维团队希望在集群中维护一个空间，以便他们可以强制实施一些严格的规程，对谁可以或谁不可以操作运行生产站点的 Pods、Services 和 Deployments 集合进行控制。
```

File: content/zh/docs/tasks/administer-cluster/namespaces-walkthrough.md [hunk 3]
```diff
@@ -0,0 +1,386 @@
...
+- janetkuo
+title: Namespaces Walkthrough
+content_template: templates/task
+--- -->
+
+{{% capture overview %}}
+<!-- Kubernetes {{< glossary_tooltip text="namespaces" term_id="namespace" >}}
+help different projects, teams, or customers to share a Kubernetes cluster. -->
+
+Kubernetes {{< glossary_tooltip text="namespaces" term_id="namespace" >}}
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply content/zh/docs/tasks/administer-cluster/namespaces-walkthrough.md]: absolutely good point 👍

**Comment by markthink** [comment]
@yyqqing  需要更新一下

**Comment by yyqqing** [comment]
@markthink 已更新。Thanks @sftim @chenrui333

**Comment by chenrui333** [comment]
/lgtm
/approve

Thanks!!

**Pull request merged**